pravega-cli/src/main/java/io/pravega/tools/pravegacli/commands/admin/PasswordFileCreatorCommand.java

```diff
@@ -44,9 +44,8 @@ public void execute() throws IOException, InvalidKeySpecException, NoSuchAlgorit
         }
     }
 
-
     public static CommandDescriptor descriptor() {
-        return new CommandDescriptor(COMPONENT, "create-password-file", "Generates file with password according to filename and credential given as argument.");
+        return new CommandDescriptor(COMPONENT, "create-password-file", "Generates file with password according to filename and credential given as argument.",new ArgDescriptor("filename", "Name of the file generated by the command"),new ArgDescriptor("credentials", "username according to which password is generated"));
```

Comment:
Please, add missing spaces after commas. And keep consistency of starting the description of command with capitals (or not), based on existing commands.